app/allocation/controllers/index.js

```diff
@@ -1,4 +1,5 @@
 const CancelController = require('./cancel')
+const CancelMoveController = require('./cancel-move')
```

Comment:
We seem to call it removing a move elsewhere. Is it worth calling this `RemoveMove` to differentiate it fully from `Cancel` (which is for the allocation itself, right?) (And yes, `RemoveMove` is kind of weird)